### ISSUE TITLE
feature request : Good hamburger menu for UWP apps

### ISSUE THREAD

**Issue by smouligneauxmw** [UXDivers/Grial-UI-Kit-Support-And-Issues#11]
I am waiting for your next version for Xamarin.Forms 2 (and UWP apps) and I hope you will offer a good hamburger menu like Android and iOs. We have to develop it manualy for the moment (with Splitview + panel + add swipe gesture  etc..) and it would be very cool to have this functionality in your UI KIT (to win time). This menu is used in all the new Microsoft Apps on Windows 10 Mobile like Weather app,, Store App etc.. (without swipe gesture but I think it would be better with). Thank you and Happy New Year ;-)

Add adaptive functionalities :
https://channel9.msdn.com/Shows/Inside-Windows-Platform/UWP-Layout-using-SplitView-and-RelativePanel-in-C-with-XAML
http://blogs.u2u.be/diederik/post/2015/07/28/A-lap-around-Adaptive-Triggers.aspx

Add Swipe gesture to open splitview pane in UWP apps : 
http://stackoverflow.com/questions/32108362/uwp-swipe-to-open-splitview-pane

**Comment by Laumania** [comment]
Great idea, but isn't this more something the Xamarin.Forms team should create?

I mean, Xamarin.Forms have this 'MasterDetailPage', which render as the "Hamburger menu" on iOS and Android, to my knowledge. 
Following the idea of Xamarin.Forms in generel, it should render to "what match the standard master-detail scenario on UWP".

Haven't taken my app to UWP yet either, as I'm awaiting GRIAL UI KIT, but you could check to see if the Hamburger menu was already there: https://developer.xamarin.com/guides/xamarin-forms/windows/getting-started/universal/

:)

**Comment by smouligneauxmw** [comment]
I think this is political. Microsoft does not provide hamburger menu "ready to use" and Xamarin met Microsoft directives not to propose a burger menu such as Android and iOS. So, we are forced (including Microsoft) to develop a custom Hamburger menu to satisfy our users (that's probably why it is not always the same hamburger menu in all the Windows 10 mobile apps).

**Comment by Laumania** [comment]
Ok, fair enough, I didn't knew that. But UWP most be handling the Master/Detail scenario in some other way then?

**Comment by smouligneauxmw** [comment]
Yes, look at the links in my first message. They improve the Splitview control. 
That's why, it would be interesting that GRIAL UI Kit propose a good Hamburger Menu ;-)  "The force would be with them".
